maven-jxr-plugin/src/main/java/org/apache/maven/plugin/jxr/AbstractJxrReport.java

```diff
@@ -557,54 +558,55 @@ protected void executeReport( Locale locale )
      * Determine the templateDir to use, given javadocTemplatesVersion
      *
      * @return
+     * @throws org.apache.maven.reporting.MavenReportException if javadocTemplatesVersion cannot be parsed
      */
     private String getTemplateDir()
+        throws MavenReportException
```

Comment:
I don't see the reason for this exception. Looking at https://github.com/codehaus-plexus/plexus-languages/blob/master/plexus-java/src/main/java/org/codehaus/plexus/languages/java/version/JavaVersion.java#L71 I can't find a way to get a NumberFormatException

Reply:
Nope, you've got a point. The Integer.parseInt() *could* throw an exception, but the regex matcher ensures the string is all digits :-)